src/components/Conversation.js

```diff
@@ -616,6 +621,40 @@ class Conversation extends Component<Props, State> {
       typingStyles.push(styles.whiteText)
     }
 
+
+    const channelPeerHasNoStake = data.conversation.peers.find(p => !p.profile.hasStake)
+    console.log('ChannelPeerHasNoStake: ', channelPeerHasNoStake)
+    const missingStakeMessage = `Participants that haven't staked will not receive messages`
```

Comment:
If the error message is static it's better to define it at the module level rather than in every render, or if it's only used in one place it can be set there rather than defining an extra constant.